### PR TITLE
Update CSV and add changelog entries

### PR DESCRIPTION
**Pull request by bjankord**

### CODE REVIEW

File: src/CernerOneIcons-CSV.csv [hunk 1]
```diff
@@ -80,7 +80,7 @@ doNotDisturb,src/icons/0039_minus_i_heavyCircleRed.svg,,
 close,src/icons/0041_x_a.svg,themeable ,
 clear,src/icons/0041_x_a.svg,themeable ,
 incomplete,src/icons/0041_x_i_heavyCircle.svg,themeable ,
-incomplete_low_light,src/icons/LL_0041_x_i_heavyCircle.svg,,
+incomplete_low_light,src/icons/LL_0041_x_i_heavyCircleWhite.svg,,
```
Comment (bjankord) [inline src/CernerOneIcons-CSV.csv]: Not sure if this is the right file. Currently, the CSV file points to `src/icons/LL_0041_x_i_heavyCircle.svg`, but that file doesn't exist. I've updated CSV to poing towards `src/icons/LL_0041_x_i_heavyCircleWhite.svg`, but maybe the `LL_0041_x_i_heavyCircle.svg` file was correct and the SVG just needs to be added?

Comment (Slothjams) [reply src/CernerOneIcons-CSV.csv]: Ended up removing incomplete_low_light altogether as it wasn't needed.

Comment (bjankord) [reply src/CernerOneIcons-CSV.csv]: Ah ok, I've removed this entry from the CSV file.

**Comment by Slothjams** [comment]
@cg5037 Do you want me to cut a tag for this or should I wait for you to review?

**Pull request merged**